components/inputs/input.css

```diff
@@ -0,0 +1,45 @@
+/* Styling for input components group */
+.input-group {
+  margin: var(--scale-sm);
+  padding: var(--scale-sm);
+}
+
+/* Common Styling for input boxes */
+.input-group input[type="text"],
+input[type="password"],
+input[type="email"],
+input[type="tel"],
+input[type="search"],
+input[type="url"] {
+  background-color: var(--default-bg-color);
+  border: 1px solid var(--secondary-hover);
+  float: left;
```

Comment:
Any specific reason why you are using float?

Reply:
In case of content in same div I want to shift my inputs to left.